pages/dug_10/4-osfv-status.md

```diff
@@ -0,0 +1,374 @@
+---
+
+# OSFV Stats
+
+---
+
+# <center> Releases? </center>
+
+We have decided on switching to a rolling release for now
+
+<center><img src="/img/dug_10/osfv-status/main-branch.png" width=250/></center>
+
+---
+
+# <center> PR stats </center>
+### <center> open-source-firmware-validation </center>
+#### <center> Total </center>
+
+<center>
+<img src="/img/dug_10/osfv-status/dasharo_prs_osfv_total.png" width=600/>
+</center>
+
+---
+
+# <center> PR stats </center>
+### <center> open-source-firmware-validation </center>
+#### <center> Difference </center>
+
+<center>
+<img src="/img/dug_10/osfv-status/dasharo_prs_osfv_diff.png" width=600/>
+</center>
+
+---
+
+# <center> PR stats </center>
+### <center> osfv-scripts </center>
+#### <center> Total </center>
+
+<center>
+<img src="/img/dug_10/osfv-status/dasharo_prs_osfv_cli_total.png" width=600/>
+</center>
+
+---
+
+# <center> PR stats </center>
+### <center> osfv-scripts </center>
+#### <center> Difference </center>
+
+<center>
+<img src="/img/dug_10/osfv-status/dasharo_prs_osfv_cli_diff.png" width=600/>
+</center>
+
+---
+
+# <center> Test modules stats </center>
+#### <center> Total </center>
+
+<center>
+<img src="/img/dug_10/osfv-status/test_counts.png" width=600/>
+</center>
+
+---
+
+# <center> Test modules stats </center>
+#### <center> Difference </center>
+
+<center>
+<img src="/img/dug_10/osfv-status/test_counts_difference.png" width=600/>
+</center>
+
+Total count increased by **415**
+
+---
+
+# <center> New test cases </center>
+
+* Added FBT suite testing the fast boot feature
+* Added ACPI suite testing ACPI drivers
+* Added USC suite testing the Always On USB feature
+* Added multiple benchmark test suites:
+  + UPP, CPP, ETHPERF, DIO
+  + testing the CPU, Ethernet and disks performance
+* Automated the CBO - Custom Boot Order test suite
+* Automated most of the AUD tests
+* Extended NVM suite with an NVMe on x2 PCIe test
+* Extended TPM suite with tests for the TPM Physical Presence Interface and changing the Endorsement Primary Seed
+* Extended UTC Docking station tests with variants for every docking station, ME enabled/disabled
+
+<!--
+Graph showing differences since the last status presentation.
+List which suites were updated and short description, like
+"new suite" or "4 new cases"
+-->
+
+---
+
+# <center> New platforms </center>
+
+* Protectli VP2440
+<!--
+Graphs probably won't be necessary.
+List new supported platforms if any
+-->
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+---
+
+# Key changes
+
+---
+
+# <center> Fedora support </center>
+
+Most test cases are also possible run on Fedora.
+The test ID naming convention has changed
+[Tests naming convention](https://github.com/Dasharo/open-source-firmware-validation/blob/develop/docs/tests-naming-convention.md)
+
+`<suite_id><case_id>.<environment_id>`
+* `suite_id` - It's typically 3 to 4 letters, uppercase, identifies the test suite. Should be related to what the test suite accomplishes.
```

Comment:
I would keep these descriptions shorter in a slide, we do not need full sentences here, and would explain it more when speaking